lightrun-broker/conf/default.template

```diff
@@ -1,14 +1,66 @@
-stream {
-    log_format basic '$remote_addr [$time_local] '
-                 '$protocol $status $bytes_sent $bytes_received '
-                 '$session_time';
+http {
     upstream lightrun {
-        server ${LIGHTRUN_SERVER}:443;
+        zone lightrun 64k;
+        server ${LIGHTRUN_SERVER}:443 max_fails=0;
+        keepalive 2;
     }
 
     server {
-        access_log /dev/stdout basic;
-        listen 443;
-        proxy_pass lightrun;
+      listen 8443 ssl;
+      server_name _ ; 
+      ssl_certificate          /etc/nginx/conf.d/certs/tls.crt;
+      ssl_certificate_key      /etc/nginx/conf.d/certs/tls.key;
+      ssl_prefer_server_ciphers on;
+
+      location / {
+        client_body_buffer_size    512k;
+        client_max_body_size       25m;
+
+
+        proxy_connect_timeout                   90s;
+        proxy_send_timeout                      90s;
+        proxy_read_timeout                      90s;
+
+        proxy_buffering                         on;
+        proxy_buffer_size                       1m;
+        proxy_buffers                           4 1m;
+        
+        proxy_max_temp_file_size                1024m;
+        
+        proxy_request_buffering                 on;
+        proxy_http_version                      1.1;
+        
+        proxy_cookie_domain                     off;
+        proxy_cookie_path                       off;
+        
+        proxy_pass https://lightrun;
```

Comment:
shouldn't this be configurable?

Reply:
`lightrun` here is the name of the `upstream`  
which is configurable from env var
```
    upstream lightrun {
        zone lightrun 64k;
        server ${LIGHTRUN_SERVER}:443 max_fails=0;
        keepalive 2;
    }
```